src/main/java/gov/usgs/ngwmn/model/MediationType.java

```diff
@@ -24,4 +25,27 @@ public static String validMediations() {
 		}
 		return validValues;
 	}
+	
+	/**
+	 * AboveDatum and Ascending are equal sort
+	 * BelowLand and Descending are equal sort
+	 * @param obj instance to compare
+	 * @return true if equal
+	 */
+	public boolean equal(Object obj) {
+		if (obj == null || !(obj instanceof MediationType)) {
+			return false;
+		}
+		if (this.equals(obj)) {
+			return true;
+		}
+		if (BelowLand.equals(this) || DESCENDING.equals(this)) {
+			return (BelowLand.equals(obj) || DESCENDING.equals(obj));
+		}
+		// ASCENDING seems more natural sort for non-well data and is default for this reason
+		if (AboveDatum.equals(this) || ASCENDING.equals(this) || DEFAULT.equals(this)) {
+			return (AboveDatum.equals(obj) || ASCENDING.equals(obj) || DEFAULT.equals(obj));
+		}
+		return false;
+	}
```

Comment:
Unlike equals() this equal() method is for checking if the instance is the same sort order. For non-groundwater data, the default sort will be ascending thus the equivalence above. This is intended to be used for other water data in the future.

Reply:
I really like your suggestion of using `equalSortOrder` as the method name!